R/ROCm/HSARuntime/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+using BinaryBuilder, Pkg
+
+name = "HSARuntime"
+version = v"7.0.01120251130"
+
+sources = [
+    FileSource("https://rocm.nightlies.amd.com/v2/gfx1150/rocm_sdk_core-7.11.0a20251130-py3-none-linux_x86_64.whl",
```

Comment:
Why use the nightlies? (I have no knowledge of ROCm releases/distribution, so just a basic overview of how ROCm stuff is released/distrbuted would be helpful)